webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -315,6 +335,7 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   private query(opts?) {
+    const trackPerformance = this.performanceService.track('contact-list:query');
```

Comment:
How about 'contact-list:query: execute_search' 

Also should we move it to the [search](https://github.com/medic/cht-core/blob/10ff96d98d8d3bdd1dfa97a47ae82f0403b4a105/webapp/src/ts/modules/contacts/contacts.component.ts#L431C3-L431C13) method which performs the actual search ?

Reply:
> How about 'contact-list:query: execute_search'

Done

> Also should we move it to the search method which performs the actual search?

That function has 2 paths one executes the query almost right away, the other does a navigation, which I think ends up trigger the search from the ngoninit, and it already have a track: `contact-list:initial_load`